### PR TITLE
Matched import functions and model inputs with new structure

### PR DESCRIPTION
**Pull request by JBHilton**

### CODE REVIEW

File: examples/simple/run.py [hunk 1]
```diff
@@ -7,17 +7,18 @@
...
 
-model_input = TwoAgeModelInput(DEFAULT_SPEC)
-
 # List of observed household compositions
 composition_list = array(
     [[0, 1], [0, 2], [1, 1], [1, 2], [2, 1], [2, 2]])
 # Proportion of households which are in each composition
 comp_dist = array([0.2, 0.2, 0.1, 0.1, 0.1,  0.1])
 
+model_input = TwoAgeModelInput(DEFAULT_SPEC, composition_list, comp_dist)
```
Comment (robertsawko) [inline examples/simple/run.py:20]: I am still a little surprised we need `TwoAgeModelInput` class.

File: model/imports.py [hunk 2]
```diff
@@ -16,19 +16,19 @@ def import_model_from_spec(spec, det):
...
 
 
 class NoImportModel(ImportModel):
-    def detected(self, t):
-        return 0.0
-
-    def undetected(self, t):
-        return 0.0
+    def cases(self, t):
+        return zeros(self.no_entries,)
```
Comment (robertsawko) [inline model/imports.py:31]: Is that comma at the end necessary?

**Review by robertsawko — APPROVED** [review]
Looks good to me - some minor comments in line.

File: examples/simple/run.py (hunk 1 above)
Comment (JBHilton) [reply examples/simple/run.py:20]: We won't actually be using the TwoAgeModelInput class anywhere in the paper version of the code, but it needs to be there for the test to work. I will be writing a new test once it's done. I haven't implemented the new specs/inputs in the simple example, but I will do on the next push.

File: model/imports.py (hunk 2 above)
Comment (JBHilton) [reply model/imports.py:31]: I will defer to your Python knowledge here and say probably not.

**Pull request merged**